src/main.go

```diff
@@ -1,22 +1,56 @@
 package main
 
-import  (
+import (
 	"flag"
-	"github.com/Agile-Arch-Angels/decentralized-tech-radar_dev/SpecReader"
+	"fmt"
+	"os"
+	"strings"
+
+	"github.com/Agile-Arch-Angels/decentralized-tech-radar_dev/Fetcher"
 	html "github.com/Agile-Arch-Angels/decentralized-tech-radar_dev/HTML"
+	"github.com/Agile-Arch-Angels/decentralized-tech-radar_dev/SpecReader"
 )
 
 func main() {
+	var repos []Fetcher.Repo
+
+	fetch := flag.String("fetch", "", "This command will fetch files from a list of git repos.\n \nHere is an example\n \n go run main.go --fetch \"https://github.com/Agile-Arch-Angels/decentralized-tech-radar_dev main ./Fetcher/something.txt https://github.com/JonasSkjodt/CopenhagenBuzz master ./Fetcher/something.txt https://gitlab.com/nagyv-gitlab/gitops-test master ./Fetcher/something.txt\"")
```

Comment:
I'm unsure if the description should be this long? I thought that the command usage example might be something that'd show up when typing `-h` or `--help` (future feat maybe, depends on the Cobra impl.) and under the feature's documentation? :)

Reply:
You are correct in this. That is too long for a small help annotation. Would suggest we fix this when we move to cobra since this will more than likely be deleted when we do so anyways 👍

Reply:
Just to add to this; I figure it's also easier, since Cobra will help with auto-generated help messages:D